toxcore/tox.api.h

```diff
@@ -286,7 +286,10 @@ const FILE_ID_LENGTH              = 32;
  */
 const MAX_FILENAME_LENGTH         = 255;
 
-
+/**
+ * Max legnth of names queried by toxcore
+ */
+const MAX_QUERY_NAME_LENGTH       = 255;
```

Comment:
ah right byte to char conversion, stupid me... Ok seems nice that way, but might be an issue for Chinese as pointed out by @initramfs